README.md

```diff
@@ -47,34 +81,59 @@ These sections describe requirements for using this module.
 
 The following dependencies must be available:
 
-- [Terraform][terraform] v0.12
-- [Terraform Provider for GCP][terraform-provider-gcp] plugin v2.0
+* [Terraform][terraform] v0.12
+* [Terraform Provider for GCP][terraform-provider-gcp] plugin v2.0
+
+### Permissions
 
-### Service Account
+A service account or user account needs the following roles to provision the
+resources of this module:
 
-A service account with the following roles must be used to provision
-the resources of this module:
+#### Google Cloud IAM roles
 
-- Storage Admin: `roles/storage.admin`
+* Service Usage Consumer: `roles/serviceusage.serviceUsageConsumer` on the
+    billing project
+* Organization Viewer: `roles/resourcemanager.organizationViewer` if using
+    `domain` instead of `customer_id`
 
 The [Project Factory module][project-factory-module] and the
-[IAM module][iam-module] may be used in combination to provision a
-service account with the necessary roles applied.
+[IAM module][iam-module] may be used in combination to provision a service
+account with the necessary roles applied.
+
+#### Google Workspace (formerly known as G Suite) roles
+
+* [Group Admin role](https://support.google.com/a/answer/2405986?hl=en)
```

Comment:
Some additional requirements 
```
To use the Groups API, you must have Google Groups for Business enabled for your domain and allow end users to create groups
```
are noted here: https://cloud.google.com/identity/docs/how-to/setup

Reply:
Added below in the APIs section.